### PR TITLE
mention .ebignore and .gitignore

### PR DESCRIPTION
**Pull request by svenfuchs**

### CODE REVIEW

File: user/deployment-v2/providers/elasticbeanstalk.md [hunk 1]
```diff
@@ -22,4 +22,9 @@ deploy:
 ```
 {: data-file=".travis.yml"}
 
+## Controlling which files are included in the zip archive
```
Comment (Pezi777) [inline user/deployment-v2/providers/elasticbeanstalk.md]:
```suggestion
## Controlling which files are included in the .zip archive
```

File: user/deployment-v2/providers/elasticbeanstalk.md [hunk 2]
```diff
@@ -22,4 +22,9 @@ deploy:
...
+
+You can control which files are included in the zip archive you upload with
```
Comment (Pezi777) [inline user/deployment-v2/providers/elasticbeanstalk.md]:
```suggestion
You can control which files are included in the `.zip` archive you upload with
```

Comment (svenfuchs) [reply user/deployment-v2/providers/elasticbeanstalk.md]: makes sense 👍

File: user/deployment-v2/providers/elasticbeanstalk.md (hunk 1 above)
Comment (svenfuchs) [reply user/deployment-v2/providers/elasticbeanstalk.md]: Personally I think "zip archive" is the right technical term, while ".zip" refers to the file extension?

Comment (svenfuchs) [reply user/deployment-v2/providers/elasticbeanstalk.md]: Personally I think "zip archive" is the right technical term, while ".zip" refers to the file extension?

Comment (svenfuchs) [reply user/deployment-v2/providers/elasticbeanstalk.md]: AWS seems to use `ZIP file`, so maybe let's follow that?

**Comment by Pezi777** [comment]
@svenfuchs thank you :-)

**Pull request merged**